ModelVale/ModelVale/CustomComponents/DropDownTextField.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DropDownTextField.swift
+//  ModelVale
+//
+//  Created by Chaytan Inman on 7/17/22.
+//
+
+import UIKit
+import iOSDropDown
+
+// Source: https://github.com/jriosdev/iOSDropDown
+@objc class DropDownTextField: DropDown {
+
+    @objc func initProperties(options: [String]) {
+        self.optionArray = options
+        print("DropDownTextField options: ")
+        print(self.optionArray)
+        self.isSearchEnable = true
```

Comment:
isSearchEnabled ? Missing a d at the end

Reply:
No the api is just weird and that is what the property is called